src/theory/bv/bitblast/aig_bitblaster.cpp

```diff
@@ -380,7 +382,7 @@ void AigBitblaster::convertToCnfAndAssert() {
 
   // // free old network
   // Abc_NtkDelete(currentAigNtk());
```

Comment:
Hmm, I guess technically, we are leaking `s_abcAigNetwork` right now. Maybe something to consider in a future PR. Also, does `s_abcAigNetwork` really need to be static? It doesn't really seem to be accessed outside of the AIG bitblaster.